src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -113,7 +113,7 @@ public function setId(string $id)
     public function startEvent(string $name, ?string $category)
     {
         if (!isset($this->events[$name])) {
-            $this->events[$name] = new StopwatchEvent($this->origin ?: microtime(true) * 1000, $category, $this->morePrecision);
+            $this->events[$name] = new StopwatchEvent($this->origin ?: microtime(true) * 1000, $category, $this->morePrecision, $name);
```

Comment:
instead, would it make sense to have $category default to $name?

Reply:
yes, sure we can make it like that.

But if I understand you correctly you want to remove the `$name` property from the `StopwatchEvent`!
If so this will not be good solution as the same problem will happen when we do like this:
```php
$stopwatch = new Stopwatch();
$stopwatch->start('name', 'cat');
$event = $stopwatch->stop('name');
echo (string) $event;
```
this will print `cat: 2.00 MiB - 0 ms` instead we need it to print the name so `name: 2.00 MiB - 0 ms`.

Otherwise, we can go with the 2nd solution suggested in the main issue #37627:
```
Rename $category to $label (to signify that this is not meant to "categorize" anything, but merely the string you will see again in the output), and $name to $id (to signify that this is just an internal ID you won't see again)
```
what do you think?!

Reply:
Re-reading the issue, I think the 2nd solution might be indeed better.

Reply:
Thank you @fabpot, I will update the PR with the required changes to apply the 2nd solution.
I'm also feeling more comfortable to the 2nd solution as well.

Reply:
Updated